frame/bags-list/src/list/tests.rs

```diff
@@ -712,7 +719,7 @@ mod bags {
 
 		ExtBuilder::default().build_and_execute_no_post_check(|| {
 			// when inserting a duplicate id of the head
-			let mut bag_1000 = Bag::<Runtime>::get(1_000).unwrap();
+			let mut bag_1000 = Bag::<Runtime>::get(1000).unwrap();
```

Comment:
Slightly against these changes, but not a big deal

Reply:
yeah I totally nuked the diff, regret, reverted it.